### PR TITLE
Fixes #9142: Horror form force airlock open finally works, horror form lings are finally voretastic

### LINKED ISSUE #9142: Horror form bugs
(WEB REPORT BY: str4t0sss REMOTE: 158.69.55.2:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 94b2b317898edbd76cbcf8497dbf00b8e981ca68 
> 
> General description of the issue
> Horror forms airlock forcing doesnt work, it dies to bleeding (not sure if intended), cannot devour humans like Aliens, but its written it can. 
> 
> What you expected to happen
> Forcing airlocks, devouring people
> 
> What actually happened
> Having to use ID's, beheading people
> 
> Steps to reproduce if possible
> Turn into horror form as a changeling, try to force a airlock or devour a mob. Get shot by a mateba and bleed out slowly.

### PR DESCRIPTION
**Pull request by ghost**
Horror form changelings can finally eat people like the ability description says you're supposed to be able to. Not sure why you'd want to, but it's there.
Removed the broken "force airlock" verb, you now force airlocks open by clicking on them with an empty hand. The effect is still the same, namely: you still can't force open airlocks that are bolted or welded, which is kinda shitty because you can punch down walls anyways.

Horror form can also force open firelocks as if with a crowbar. Also fixes unrelated bug that didn't let you open firelocks with TK (remaining part of  #12894).

Also added a shitty sound I shittily whipped up in shitty audacity.

:cl:
 * bugfix: Fixed horror form changelings not being able to force airlocks open. To do so, click them with an open hand. This still won't work on bolted/welded airlocks, though.
 * bugfix: Fixed horror form changelings not being able to eat people.

### CODE REVIEW

File: code/game/machinery/doors/door.dm [hunk 1]
```diff
@@ -170,8 +170,29 @@ var/list/all_doors = list()
 		else
 			return open()
 
+	if(horror_force(user))
+		return
+
 	denied()
 
+/obj/machinery/door/proc/horror_force(var/mob/living/carbon/human/H) //H is for HORROR, BABY!
```
Comment (PJB3005) [inline code/game/machinery/doors/door.dm:178]: Put this on `/obj/machinery/door/airlock`

File: code/game/machinery/doors/door.dm [hunk 2]
```diff
@@ -170,8 +170,29 @@ var/list/all_doors = list()
...
+	if(!(ishorrorform(H)))
```
Comment (PJB3005) [inline code/game/machinery/doors/door.dm]: Redundant parentheses.

File: code/game/machinery/doors/door.dm [hunk 3]
```diff
@@ -170,8 +170,29 @@ var/list/all_doors = list()
...
+		return
```
Comment (PJB3005) [inline code/game/machinery/doors/door.dm]: Make this return `FALSE` explicitly.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/game/machinery/doors/door.dm (hunk 1 above)
Comment (9600bauds) [reply code/game/machinery/doors/door.dm:178]: Negative, `/obj/machinery/door/firedoor/` is not a child of airlock

Comment (PJB3005) [reply code/game/machinery/doors/door.dm:178]: Right.

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**